File: algorithms/javascript/sorting/selectionSort.js
const selectionSort = arr => {
    for (let i = 0; i < arr.length; i++) {
        let min_idx = i;
        for (let j = i + 1; j < arr.length; j++)
            if (arr[j] < arr[min_idx])
                min_idx = j;
         if (min_idx != i) {
             // Swapping the elements
             let tmp = arr[i];
             arr[i] = arr[min_idx];
             arr[min_idx] = tmp;
        }
    }
    return arr;
}

console.log(selectionSort([5, 2, 4, 6, 1, 3]));

File: algorithms/Python/sorting/selectionSort.py
def selectionSort(arr):
    for i in range(len(arr)):
        min_indx = i
        for j in range(i + 1, len(arr)):
            if arr[min_indx] > arr[j]:
                min_indx = j

        arr[i], arr[min_indx] = arr[min_indx], arr[i]

A = [64, 25, 12, 22, 11]
selectionSort(A)
for i in range(len(A)):
    print("%d" %A[i]),
